In [1]:
import pandas as pd 
import numpy as np 
import matplotlib
import seaborn

In [3]:
inven_path='Data\\Inventory Dataset.xlsx'
new_inv_path='Data\\New Inventory.xlsx'

In [4]:
inventory_df=pd.read_excel(inven_path)
new_inventory_df=pd.read_excel(new_inv_path)

## Inventory Dataset.xlsx

In [17]:
print("The Inventory dataset contains {} rows and {} columns.".format(inventory_df.shape[0],inventory_df.shape[1]))


the inventory dataset contains 356 rows  and 3 columns.


In [19]:
len(inventory_df['Item Number'].unique())

356

## New Inventory.xlsx

In [18]:
print("The New inventory dataset contains {} rows and {} columns.".format(new_inventory_df.shape[0],new_inventory_df.shape[1]))

The New inventory dataset contains 1039 rows and 3 columns.


In [14]:
len(new_inventory_df['Item Number'].unique())

213

In [15]:
len(new_inventory_df['Inventory Receive'].unique())

23

The inventory data set contains 356 unique item numbers and the number of items in that inventory. The new inventory dataset contains 1,039 rows of numbers <br> 
showing how many new items were added to inventory and when they arrived. The new inventory dataset contains 213 unique items, which means that out of 356 items,<br>
only 213 items were received on 23 different unique dates.